### PR TITLE
FCBHDBP-518 uploader (python) - recognize Deuterocanon books and sequence accordingly

### PR DESCRIPTION
**Pull request by vichugofsl**
## Description
It has removed the current logic for the Deuterocanonical books and the additional chapter map array that had been created to include them. Going forward, it will retrieve the chapter map values from databases.

## NOTE 1
We need to update the following bible_books records to support the current changes:

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-518

```sql
UPDATE books SET chapters = 1 WHERE id = "INT" and chapters is NULL;
UPDATE books SET chapters = 2 WHERE id = "BAK" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "LXX" and chapters is NULL;
UPDATE books SET chapters = 2 WHERE id = "CNC" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "TDX" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "NDX" and chapters is NULL;
UPDATE books SET chapters = 5 WHERE id = "OTH" and chapters is NULL;
UPDATE books SET chapters = 4 WHERE id = "XXA" and chapters is NULL;
UPDATE books SET chapters = 3 WHERE id = "XXB" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "XXC" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "XXD" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "XXE" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "XXF" and chapters is NULL;
UPDATE books SET chapters = 1 WHERE id = "XXG" and chapters is NULL;
UPDATE books SET chapters = 6 WHERE id = "FRT" and chapters is NULL;
UPDATE books SET chapters = 26 WHERE id = "GLO" and chapters is NULL;
```
## How Do I QA This
- I have executed the following command on my local environment:

```shell
python3 load/DBPLoadController.py test s3://etl-development-input BULBOBO1ET
```

```shell
python3 load/FilenameParser.py test /home/vichugof/fcbh-etl-local/ ENGWEBO2ET
```

### CODE REVIEW

File: load/FilenameParser.py [hunk 1]
```diff
@@ -488,12 +488,6 @@ def __init__(self, config):
 	def process3(self, filesets):
 		db = SQLUtility(self.config)
 		self.chapterMap = db.selectMap("SELECT id, chapters FROM books", None)
-		## I am not certain the LXX actually exists
-		extras = {"FRT":6, "INT":1, "BAK":2, "LXX":1, "CNC":2, "GLO":26, "TDX":1, "NDX":1, "OTH":5, 
-			"XXA":4, "XXB":3, "XXC":1, "XXD":1, "XXE":1, "XXF":1, "XXG":1}
-		self.chapterMap.update(extras)
-		corrections = {"MAL":3, "MAN":1, "PS2":1, "BAR":5, "EZA":9, "SIR":52}
-		self.chapterMap.update(corrections)
```
Comment (bradflood) [inline load/FilenameParser.py:496]: let's also remove maxChapterMap update (for JOL, PSA, MAL and BAR) and usfx2Map update

**Review by bradflood — CHANGES_REQUESTED** [review]

File: load/FilenameParser.py (hunk 1 above)
Comment (vichugofsl) [reply load/FilenameParser.py:496]: @bradflood I have removed the changes for `JOL, PSA, MAL and BAR` and the changes for the usfx2Map list

File: load/FilenameParser.py [hunk 2]
```diff
@@ -488,19 +488,8 @@ def __init__(self, config):
...
 		self.maxChapterMap = self.chapterMap.copy()
-		corrections = {"JOL":4, "PSA":151, "MAL":4, "BAR":6}
-		self.maxChapterMap.update(corrections)
 		self.usfx2Map = db.selectMap("SELECT id_usfx, id FROM books", None)
-		extras = {"FR":"FRT", "IN":"INT", "BK":"BAK", "CN":"CNC", "GS":"GLO", "TX":"TDX", "OH":"OTH",
-			"XA":"XXA", "XB":"XXB", "XC":"XXC", "XD":"XXD", "XE":"XXE", "XF":"XXF", "XG":"XXG"}
-		self.usfx2Map.update(extras)
 		self.usfx2Map["J1"] = "1JN" ## fix incorrect entry in books table
```
Comment (bradflood) [inline load/FilenameParser.py:493]: sorry... please also remove this usfx2Map correction.

**Review by bradflood — CHANGES_REQUESTED** [review]

File: load/FilenameParser.py (hunk 2 above)
Comment (bradflood) [reply load/FilenameParser.py:493]: also, for the code that is removed, please add DML that I can run on NEWDATA (DML = Data Manipulation Language.. the update commands to change the DB associated with the code we are removing)

**Review by bradflood — CHANGES_REQUESTED** [review]

File: load/FilenameParser.py (hunk 2 above)
Comment (vichugofsl) [reply load/FilenameParser.py]: @bradflood I have removed this line as well and I have updated the PR description to add the DML (for our case I have created the updates statements)

**Review by bradflood — APPROVED** [review]

**Pull request merged**